lib/fa-harness-tools/check_forward_deploy.rb

```diff
@@ -23,21 +24,27 @@ def verify?
       @logger.start
       @logger.context_info(@client, @context)
 
-      current_tag = @client.last_deploy_tag(
-        prefix: @tag_prefix, environment: @context.environment)
+      if @tag_prefix
+        current_tag = @client.last_deploy_tag(
+          prefix: @tag_prefix, environment: @context.environment)
```

Comment:
Am i right in thinking you are swapping the inputs to this so that it is no longer passed a tag_prefix (this is just the 'old' path supported still? )

And you are updating the input to call this with --current-sha?

Reply:
Yer, tag_prefix will take precedence so everything continues to work as it should. Then we can switch to passing in the sha instead then delete most of the code in this function